ocaml/typing/typedtree.ml

```diff
@@ -1035,13 +1035,20 @@ let rev_let_bound_idents_full bindings =
   List.iter (fun vb -> iter_bound_idents add vb.vb_pat) bindings;
   !idents_full
 
-let let_bound_idents_with_modes_and_sorts bindings =
+let let_bound_idents_with_modes_sorts_and_checks bindings =
   let modes_and_sorts = Ident.Tbl.create 3 in
```

Comment:
I renamed the variable.  I hesitate to try to come up with a good name for a record because we already have `let_bound_idents_full` (which exists upstream, so I don't particularly want to rename it) and having `_full` and `_augmented` variants seems confusing.